algorithms/Python/strings/find_all_permutations.py

```diff
@@ -0,0 +1,29 @@
+"""
+Find all the permutations of a given string
+Sample inut: 'ABC'
```

Comment:
typo found by codespell 
inut ==> input

```suggestion
Sample input: 'ABC'
```

Reply:
I updated the file and committed the changes without the typo.